Formula/g/gssh.rb

```diff
@@ -18,12 +18,11 @@ class Gssh < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "db58030c9483c22ecde1258a00dd0bd3f1da7649e1114178e0495e0195296ef6"
   end
 
+  deprecate! date: "2025-05-25", because: :repo_archived
```

Comment:
The repo has moved to here - https://github.com/int128/gradle-ssh-plugin

Reply:
yeah, but it is no longer a standalone tool, but a gradle plugin, maybe use a different deprecation message?